src/windows/windowButtons.component.tsx

```diff
@@ -70,10 +71,21 @@ type DataRow = {
 
 export const constructDataRows = (
   XAxis: string,
-  plots: PlotDataset[]
+  plots: PlotDataset[],
+  selectedPlotChannels: SelectedPlotChannel[]
 ): (string | number)[][] => {
   // First row of file, i.e. the column names
   const headerRow = [XAxis].concat(plots.map((plot) => plot.name));
+  // First row of file with units, i.e. the column names
+  const headerRowWithUnits = [XAxis].concat(
+    plots.map((plot) => {
+      const channel = selectedPlotChannels.find(
+        (channel) => channel.name === plot.name
+      );
+
+      return `${channel?.displayName}/${channel?.name}/${channel?.units}`;
```

Comment:
Maybe something like `'Unknown channel'` should be returned if `channel` is `undefined`? Otherwise, the string could end up looking like `undefined/undefined/undefined`.